app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,31 @@
+= f.input "loan_types" do
```

Comment:
There are some parts of the form that look like they are not properly hooked up to successfully submit and save. I'll need some help with that.